src/main.tsx

```diff
@@ -67,7 +67,7 @@ Devvit.addSettings([
         }
       }
     }
-  },
+  } as ValidatedStringField,
```

Comment:
Why

Reply:
Without that it doesn't know what the type of the object is, so it doesn't know what the `setting` argument for the `onValidate` function is, and that's an error if `noImplicitAny` is `true`.

Reply:
I'm not sure why it doesn't know what the type of the object is though. It should be able to figure it out from the `type` field, but maybe there are to many indirections.

Reply:
My env can infer the value of `settings.value` from the `type` field

Reply:
![image](https://github.com/user-attachments/assets/872ac779-e27c-42e0-8640-341ea1d0c063)

Reply:
Do you have access to internal Devvit code in your environment? Tracing back the types I see that `BooleanField` is `any` for me, which causes everything that uses it to lose its `type`: `BooleanField` -> `ValidatedBooleanField` -> `SettingsFormField`.

Reply:
Nah, personal env. I wonder if one of our devvit versions are out of date. But theoretically, the `type: 'string'` should preclude it from being a `BooleanField` type

Reply:
I reinstalled everything and this isn't an issue now.